pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -533,16 +533,25 @@ type BackupStorageS3Spec struct {
 	CredentialsSecret string `json:"credentialsSecret"`
 }
 
+type BackupStorageAzureSpec struct {
+	Account           string `json:"account,omitempty"`
```

Comment:
maybe we should take account id from secret. please ask @spron-in

Reply:
Please use the following keys for secret:
```
 AZURE_STORAGE_ACCOUNT_NAME:
  AZURE_STORAGE_ACCOUNT_KEY: 
```

Reply:
Resolved in 054f92f163b58ce537da894e76dbdeba575b8e71